### ISSUE TITLE
Google Play Login

### ISSUE THREAD

**Issue by Safewaygames** [insthync/shooter-io-playfab-client-demo#7]
I do apologize for all the messages but I been working a lot with the kit and finding various bugs. Google play login with plafab is broken set up everything many time over again but keep getting google No Id Token Included in response. and according to playfab docs which was just updated the new code that needs to be added but I am not sure where it needs to be added. Maybe when you have time can take a look `https://docs.microsoft.com/en-us/gaming/playfab/features/authentication/platform-specific-authentication/google-sign-in-unity` 
`PlayGamesClientConfiguration config = new PlayGamesClientConfiguration.Builder()
.AddOauthScope("profile")
.RequestServerAuthCode(false)
.Build();
PlayGamesPlatform.InitializeInstance(config);`

thats supposed to be the new code

**Comment by insthync** [comment]
You can try it in `PlayfabAuthClient.cs`

**Comment by Safewaygames** [comment]
Yea I was looking in there but when I tried to add the code it just breaks even though I get no errors.

**Comment by Safewaygames** [comment]
I even tried the new update you posted and still gives the same error google Oath no ID token in response.

**Comment by insthync** [comment]
Did you try by create internal test release, upload an APK and download from internal test URL to test?

**Comment by Safewaygames** [comment]
yes I have all that set up already  I have an internal track App is up for pre register and we have a closed beta track all Live. we been trying everything we can about it for the last 2 weeks.
